easybuild/easyconfigs/c/cpu_features/cpu_features-0.6.0-foss-2020b.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = 'CMakeMake'
+
+name = 'cpu_features'
+version = '0.6.0'
+
+homepage = 'https://github.com/google/cpu_features'
+description = """A cross-platform C library to retrieve CPU features (such as available instructions) at runtime."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/google/cpu_features/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['95a1cf6f24948031df114798a97eea2a71143bd38a4d07d9a758dda3924c1932']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+
+modextrapaths = {'CPATH': 'include/cpu_features'}
+
+sanity_check_paths = {
+    'files': ['bin/list_cpu_features', 'lib/libcpu_features.a'],
```

Comment:
Please include a sanity check command for `list_cpu_features`?

Reply:
done